### PR TITLE
Patch for rinkeby

### PR DESCRIPTION
**Pull request by ailisp**
Still didn't observe a block without difficulty. But difficulty is not checked in eth-client rust contract, it's okay to give a reasonable default value here to work around.

### CODE REVIEW

**Comment by ailisp** [comment]
CI is expected to fail and need to pass before merge https://github.com/near/rainbow-bridge-cli/pull/337

File: eth2near-relay/index.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ function execute(command, _callback) {
 }
 
 function web3BlockToRlp(blockData) {
-  blockData.difficulty = parseInt(blockData.difficulty, 10)
+  blockData.difficulty = parseInt(blockData.difficulty || '0', 10)
```
Comment (maxzaver) [inline eth2near-relay/index.js:25]: Please add the comment that `difficulty` should be always present on PoW Ethereum networks.

**Review by maxzaver — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -51,4 +51,4 @@
     "semi": false,
     "singleQuote": true
   }
-}
+}
```
Comment (Kouprin) [inline package.json]: could you please setup your json-editor to always add \n at the end?

Comment (ailisp) [reply package.json]: I think i was set "format on save by prettier, use prettier config in the repo", let me double check

Comment (ailisp) [reply package.json]: ah i didn't turn on for json files

**Pull request merged**